docker-compose.yaml

```diff
@@ -1,20 +1,38 @@
 services:
+
   model-server:
     # Keep the version in sync with the modelix version in gradle/libs.versions.toml
-    image: modelix/modelix-model:3.11.0
+    image: modelix/model-server:3.11.0
+    profiles:
+      - uc1
```

Comment:
I would like to name the profiles, `useCase1a`, `useCase1b` and `useCase2` to make the better attributable to the use case numbers in the upcoming sample documentation.

Reply:
done